modules/cloudflaremanualinvalidator/src/Form/CloudFlareCacheClearForm.php

```diff
@@ -144,36 +118,15 @@ public function purgeEntireZone(array &$form, FormStateInterface $form_state) {
    *   The current state of the form.
    */
   public function purgePaths(array &$form, FormStateInterface $form_state) {
-    $config = $this->config('cloudflare.settings');
-    $api_key = $config->get('apikey');
-    $email = $config->get('email');
-    $zone = $config->get('zone');
-
-    try {
-      $zone_api = new ZoneApi($api_key, $email);
-
-      // @todo rethink how to handle cloudflare zones in Drupal.
-      if (is_null($zone)) {
-        $zones = $zone_api->listZones();
-        $zone = $zones[0]->getZoneId();
-      }
-      $zone_api->purgeAllFiles($zone);
-    }
-
-    catch (CloudFlareHttpException $e) {
-      drupal_set_message("Unable to clear paths. " . $e->getMessage(), 'error');
-      \Drupal::logger('cloudflare')->error($e->getMessage());
-      return;
-    }
+    $paths = $form_state->getValue('paths');
 
-    catch (CloudFlareApiException $e) {
-      drupal_set_message("Unable to clear paths. " . $e->getMessage(), 'error');
-      \Drupal::logger('cloudflare')->error($e->getMessage());
-      return;
-    }
+    // Remove extra-whitespace.
+    $paths = str_replace(' ', '', $paths);
```

Comment:
Is this really what you want/need? This seems highly suspicious to me. I suspect you actually want `trim()`?

Should have test coverage to prove that it works as expected (though I guess that's for a follow-up).